gatsby-node.js

```diff
@@ -31,7 +31,11 @@ exports.createPages = async ({ actions, graphql, reporter }) => {
     return
   }
 
-  result.data.allMarkdownRemark.edges.forEach(({ node }) => {
+  // Filter out the content that we don't want pages created for
+  const postOrPage = result.data.allMarkdownRemark.edges
+    .filter(edge => edge.node.frontmatter.template !== "announcement");
+
+  postOrPage.forEach(({ node }) => {
```

Comment:
I'll flag that this feels a liiiiittle forced, but don't have a strong idea of how to do it better atm.